SummerCamps.md

```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
 
 <div class="centered">
     <p>
```

Comment:
Why use a large <p> tag when all the contents can be done easier with md? We are in a md file not an html file.